docs/Custom_Operation_for_GPUs.md

```diff
@@ -403,7 +404,7 @@ Cell In [8], line 7
 NotImplementedError: Differentiation rule for 'rms_norm_fwd' not implemented
 ```
 
-# Differentiation rule
+### Differentiation rule
```

Comment:
## instead of ###?

Reply:
Thanks @seanprime7 ! Done.

```
## Differentiation rule
 The backward operation failed with the error `NotImplementedError: Differentiation rule for 'rms_norm_fwd' not implemented`.  It means that, although we have defined `rms_norm_fwd` and `rms_norm_bwd`, JAX doesn't know the relationship between them.
```

Reply:
Y [[](url)quie=N_le%sdi#o=peR=m<I/so=bambino<I/yo]n%0=SAT>D=Roy<al]ti#D](url) uba&y[](url)